32-bit build broken: mismatched types uint64 and uint32

```
/usr/lib/golang/pkg/tool/linux_386/compile -o $WORK/b377/_pkg_.a -trimpath "$WORK/b377=>" -shared -p github.com/containers/libpod/libpod -installsuffix shared -buildid UUzr_L8CfK4ssO0laMJr/UUzr_L8CfK4ssO0laMJr -goversion go1.13beta1 -D "" -importcfg $WORK/b377/importcfg -pack ./boltdb_state.go ./boltdb_state_internal.go ./boltdb_state_linux.go ./container.go ./container.log.go ./container_api.go ./container_commit.go ./container_graph.go ./container_inspect.go ./container_internal.go ./container_internal_linux.go ./container_linux.go ./container_log_linux.go ./container_top_linux.go ./diff.go ./events.go ./healthcheck.go ./healthcheck_linux.go ./in_memory_state.go ./info.go ./kube.go ./mounts_linux.go ./networking_linux.go ./oci.go ./oci_attach_linux.go ./oci_internal_linux.go ./oci_linux.go ./options.go ./pod.go ./pod_api.go ./pod_internal.go ./pod_top_linux.go ./runtime.go ./runtime_cstorage.go ./runtime_ctr.go ./runtime_img.go ./runtime_migrate.go ./runtime_pod.go ./runtime_pod_infra_linux.go ./runtime_pod_linux.go ./runtime_renumber.go ./runtime_volume.go ./runtime_volume_linux.go ./state.go ./stats.go ./stats_config.go ./storage.go ./util.go ./util_linux.go ./volume.go ./volume_internal.go $WORK/b377/_cgo_gotypes.go $WORK/b377/oci_attach_linux_cgo.cgo1.go $WORK/b377/_cgo_import.go
# github.com/containers/libpod/libpod
_build/src/github.com/containers/libpod/libpod/stats.go:90:17: invalid operation: cgroupLimit > physicalLimit (mismatched types uint64 and uint32)
_build/src/github.com/containers/libpod/libpod/stats.go:91:3: cannot use physicalLimit (type uint32) as type uint64 in return argument
```
full logs: [i686](https://kojipkgs.fedoraproject.org//work/tasks/9422/36629422/build.log) and [armv7hl](https://kojipkgs.fedoraproject.org//work/tasks/9421/36629421/build.log) (scroll to bottom).

--- a/libpod/stats.go
+++ b/libpod/stats.go
@@ -86,7 +86,8 @@ func getMemLimit(cgroupLimit uint64) uint64 {
 		return cgroupLimit
 	}
 
-	physicalLimit := si.Totalram
+	//nolint:unconvert
+	physicalLimit := uint64(si.Totalram)
 	if cgroupLimit > physicalLimit {
 		return physicalLimit
 	}

